app/(root)/posts/page.tsx

```diff
@@ -4,16 +4,15 @@ import Link from 'next/link';
 const Posts = () => {
   return (
     <>
-      <h1>Recent Posts</h1>
-      <Link href="/posts/1">
+      {/* <h1>Recent Posts</h1> */}
+      {/* Map through all posts */}
+      <Link className='paragraph-3-medium text-white-300' href="/posts/1">
```

Comment:
Font styles are going to be inherited. So why don't we just assign the paragraph and color styling once 

```js
    <section className="paragraph-3-medium flex flex-col text-white-300">
      {/* <h1>Recent Posts</h1> */}
      {/* Map through all posts */}
      <Link href="/posts/1">Post1</Link>
      <Link href="/posts/2">Post2</Link>
      <Link href="/posts/3">Post3</Link>
    </section>
```

You could probably just add these to your NavSection code and remove them entirely from this and Quicklink.

Reply:
cool!